README.md

```diff
@@ -53,7 +57,7 @@ npx @lingui/codemods <transform> <path> --jscodeshift="--printOptions='{\"quote\
 A CLI is built-in to help you migrate your codebase, will ask you some questions:
 
 ```sh
-➜  project git:(master) lingui-codemod
+➜  project git:(master) @lingui/codemods
```

Comment:
I thought this is the name of command from `bin/lingui-codemod` not the package name, isn't it?

Reply:
yes, you're right, it's the morning without coffee ☕  haha